GrassChoppers/Modules/Home/HomeViewController.swift

```diff
@@ -39,6 +41,8 @@ class HomeViewController: UIViewController {
             $0.centerX.equalToSuperview()
             $0.top.equalTo(label.snp.bottom)
         }
+        
+        logicController.viewDidLoad()
```

Comment:
Here, the view controller has loaded, and so we are informing the logic controller so that it may do whatever it needs to do. We know that LogicController will [request some data from DataManager](https://github.com/jwitcig/GrassChoppers/pull/2/files#diff-95819295f08c522fd5948df08c71a1bbR31), but ViewController doesn't know that....and it doesn't care (it doesn't need to, it just listens for when LogicController publishes new data).